source/iface/usm_allocations.rst

```diff
@@ -94,6 +94,199 @@ the ``device`` and ``context``.
 
 .. seealso:: `C++ Allocator <https://en.cppreference.com/w/cpp/named_req/Allocator>`__
 
+
+.. _allocation_functions :
+
+====================
+Allocation functions
+====================
+
+=======================
+``sycl::malloc_device``
+=======================
+
+::
+
+   void* sycl::malloc_device(size_t numBytes, const device& syclDevice,
+                          const context& syclContext,
+                          const property_list& propList = {});
+
+   template <typename T>
+   T* sycl::malloc_device(size_t count, const device& syclDevice,
+                          const context& syclContext,
+                          const property_list& propList = {});
+
+   void* sycl::malloc_device(size_t numBytes, const queue& syclQueue,
+                          const property_list& propList = {});
+
+   template <typename T>
+   T* sycl::malloc_device(size_t count, const queue& syclQueue,
+                          const property_list& propList = {});
+
+   void* sycl::aligned_alloc_device(size_t alignment, size_t numBytes,
+                          const device& syclDevice,
+                          const context& syclContext,
+                          const property_list& propList = {});
+
+   template <typename T>
+   T* sycl::aligned_alloc_device(size_t alignment, size_t count,
+                          const device& syclDevice,
+                          const context& syclContext,
+                          const property_list& propList = {});
+
+   void* sycl::aligned_alloc_device(size_t alignment, size_t numBytes,
+                          const queue& syclQueue,
+                          const property_list& propList = {});
+
+   template <typename T>
+   T* sycl::aligned_alloc_device(size_t alignment, size_t count,
+                          const queue& syclQueue,
+                          const property_list& propList = {})
+
+.. rubric:: Parameters
+
+==================  ===
+``alignment``       alignment of allocated data
+``numBytes``        allocation size in bytes
+``count``           number of elements
+``syclDevice``      See :ref:`device`
+``syclQueue``       See :ref:`queue`
+``syclContext``     See :ref:`context`
+``propList``
+==================  ===
+
+On success USM device allocation, these functions return a pointer to the newly
+allocated memory, which must eventually be deallocated with ``sycl::free`` in
+order to avoid a memory leak. If there are not enough resources to allocate the
+requested memory, these functions return ``nullptr``.
+
+When the allocation size is zero bytes (``numBytes`` or ``count`` is zero),
+these functions behave in a manor consistent with C++ ``std::malloc``.
```

Comment:
```suggestion
these functions behave in a manner consistent with C++ ``std::malloc``.
```

Reply:
This is a spec spelling mistake that I'll fix in the SYCL2020 spec - saying it here now so that this doesn't go out of sync

Reply:
Fixed with commit https://github.com/KhronosGroup/SYCL_Reference/pull/85/commits/55b446b98f11cf4776b52140f8ca7d8557abe5d0